src/index.html

```diff
@@ -3,11 +3,65 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="mac image"
+      >
+
+      <div class="card__info">
+        <h3 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <div class="card__code">
+          Product code: 195434
+        </div>
+
+        <div class="card__rating">
+          <ul class="stars stars--4">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+
+          <div class="card__reviews">
+            Reviews: 5
+          </div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-title">
+            Price:
+          </div>
+
+          <div class="card__price-amount">
+            $2,199
+          </div>
+        </div>
+        <button class="card__button" data-qa="hover">
```

Comment:
Add space for more readable
```suggestion
        </div>
        
        <button class="card__button" data-qa="hover">
```